src/main/java/com/iota/iri/service/dto/GetNodeAPIConfigurationResponse.java

```diff
@@ -0,0 +1,88 @@
+package com.iota.iri.service.dto;
+
+import com.iota.iri.conf.IotaConfig;
+
+/**
+ * Contains information about the result of a successful {@link com.iota.iri.service.API#getNodeAPIConfigurationStatement()} API call.
+ * See {@link com.iota.iri.service.API#getNodeAPIConfigurationStatement()} for how this response is created.
+ */
+public class GetNodeAPIConfigurationResponse extends AbstractResponse {
+    private int maxFindTransactions;
```

Comment:
Can you add links/javadoc to these fields?

`{@link IotaConfig#getMaxRequestsList()}` Would be sufficient.
Otherwise the doclet wont pick it up as a returned variable

Reply:
I added the Javadoc to the getters. Should i move them to the field declaration?

Reply:
Renamed field variable to match getter and moved javadoc to field declaration.

Reply:
Perfect